app/assets/javascripts/osem-schedule.js

```diff
@@ -86,8 +133,18 @@ $(document).ready( function() {
     snap: '.schedule-room-slot',
     revertDuration: 200,
     revert: function (event, ui) {
-        console.log(event.attr);
-        return !event;
+      console.log(event.attr);
```

Comment:
I can remove that because it is the only log that is still here and I think it is not needed.